packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -494,7 +490,7 @@ contract Deploy is Deployer {
         // are always proxies.
         Types.ContractSet memory contracts = _proxiesUnstrict();
         contracts.OptimismPortal = address(portal);
-        ChainAssertions.checkOptimismPortal({ _contracts: contracts, _cfg: cfg, _isPaused: true });
+        ChainAssertions.checkOptimismPortal({ _contracts: contracts, _cfg: cfg, _isPaused: false });
```

Comment:
Paused is no longer a property of the optimism portal but instead of the superchain registry, perhaps the check should be the paused getters return the same value